app/src/main/java/org/simlar/widgets/CallActivity.java

```diff
@@ -53,6 +55,9 @@
 {
 	private static final String INTENT_EXTRA_SIMLAR_ID = "simlarId";
 
+	private static final int PERMISSION_REQUEST_CODE_VIDEO_REQUEST = PermissionsHelper.DEFAULT_REQUEST_CODE + 1;
```

Comment:
In case these codes only need to be locally unique, I don't understand the need for the `+ 1`. Otherwise defining them here will not prevent duplicates on the global level.